_episodes/09-documentation.md

```diff
@@ -412,4 +412,45 @@ conda:
 Commit to these changes and push to your repository. You should now see your documentation strings.
 
 
-{% include links.md %}
\ No newline at end of file
+#### Automatically Generating Documentation Using Sphinx-AutoAPI
+Sphinx includes an alternative to autodoc/autosummary that will automatically generate an "API Reference" page based
+on the docstrings throughout the projet's source code upon compiling using `make html`.
+This is [Spinx-AutoAPI](https://github.com/readthedocs/sphinx-autoapi). To install Sphinx-AutoAPI, use `pip`:
+
+``pip install sphinx-autoapi``
+
+There are a few modifications necessary to use Sphinx-AutoAPI.  
+ - In the the conda-env file `devtools/conda-envs/test_env.yaml`, include `sphinx-autoapi` under
+ `dependencies`, pip-only installs  
+ - At the end of the `docs/requirements.yml` file, under `pip-only installs`, include`- sphinx-autoapi`
+ - In `docs/conf.py`, remove `sphinx.ext.autodoc` and `sphinx.ext.autosummary` and add `autoapi.extension` to the
+ `extensions` list variable. Then comment out or delete the line: `autosummary_generate = True`.
+ - Optional: remove `* :ref:'modindex'` from `index.rst`, since API Reference will show up autmoatically in
+ `.. toctree::`
```

Comment:
Would be useful to see `conf.py` with suggested changes here.

Reply:
Added in latest commit.